src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,19 +24,26 @@ class Registry
 
     /**
      * @param Workflow $workflow
-     * @param string   $className
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $className)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
@andesk yes. Your current proposal is a BC break, and cannot be accepted for Symfony 3.3 (and would not even be acceptable for Symfony 4.0 as is, as it does not allow to have a progressive upgrade path)